File: 0009-palindrome-number/0009-palindrome-number.java
class Solution 
{
    public boolean isPalindrome(int x) 
    {
      int r = 0, s=0, n=0;
      n=x;
      if(n<0)
      {
         n= n*(-1);
      }
      while(n!=0){
      r=n%10;  
      s=s*10+r;
      n=n/10;}
      if(s == x)
      {
        System.out.println(s);
        return true;
      }
      else
      {
        return false;
      }
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        s = str(x)
        l=0
        r=len(s)-1
        flag= True
        while l<r:
            if s[l]!=s[r]:
                flag=False
            l+=1
            r-=1
        return flag